pkg/scheduler/bind-plugin.go

```diff
@@ -41,6 +41,14 @@ func (p *BindPlugin) Bind(
 ) *framework.Status {
 	// we will use the pod uid as the placement request name.
 	prname := string(pod.UID)
+	if prname == "" {
+		return framework.AsStatus(fmt.Errorf("pod %v/%v is missing its UID", pod.Namespace, pod.Name))
```

Comment:
It's acceptable to expect a pod to have its UID set for the testing purposes. We might get some friction when posing this restriction upstream as the UID field is still considered as optional. Yet, I am not aware of any case where the field is empty.